official-antistasi-community/A3-Antistasi
# Phase 3 MIE implementation

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
    Moves JNL definition to mod specific files in the Templates folder
    Changes the weapon compatibility listings to arrays that can be updated by various files, instead of hard coding all weapons 
    for all vehicles.

### Please specify which Issue this PR Resolves.
closes #1278 

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in singleplayer?
2. [x] Have you loaded the mission in LAN host?
3. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
  Load all modsets and check that all JNL actions still work as intended.
********************************************************
Notes:
  Can't test weapon points as they require JNG which has been disabled for some time.
